templates/v6.yml

```diff
@@ -1,3 +1,5 @@
+image: ubuntu
```

Comment:
No need to do this, since each runner now defaults to an Ubuntu-like image that is best suited for the driver.